src/code-generator/test/test-helpers.ts

```diff
@@ -8,6 +9,22 @@ function getRandomFolderPath(basePath: string) {
   return path.join(basePath, randomFolderName);
 }
 
+export async function doesFolderExistInPath(path: string) {
+  return await fsExtra.pathExists(path);
+}
+
+export async function doesFileContainPhrase(filePath: string, phrase: string) {
+  const readStream = createReadStream(filePath, { encoding: "utf-8" });
+  for await (const chunkOfData of readStream) {
+    const chunkAsString = chunkOfData + "";
```

Comment:
Instead of converting every chunk to string maybe convert the phrase to buffer and check if the `chunkOfData` includes the `phraseBuffer`